kalite/testing/behave_helpers.py

```diff
@@ -239,9 +243,33 @@ def build_url(context, url):
     return urljoin(context.config.server_url, url)
 
 def go_to_homepage(context):
+    """
+    Goes to the homepage. Raises a TimeoutException if page does not load.
+    """
     url = reverse("homepage")
-    context.browser.get(build_url(context, url))
+    _go_to_url(context, build_url(context, url))
+
+def go_to_coachreports(context):
+    """
+    Goes to coach reports. Raises a TimeoutException if page does not load.
+    """
+    url = reverse("coach_reports")
+    _go_to_url(context, build_url(context, url))
 
+def _go_to_url(context, url, wait_time=MAX_PAGE_LOAD_TIME):
+    """
+    Tries to browse to a page. Waits for the page to load, since WebDriver.get does not.
```

Comment:
What I think is happening is this. First of all, scenarios within a feature share one WebDriver instance. One of the failing steps is this:
```
When I click on the Show Tabular Report button     # kalite/coachreports/features/steps/coachreports.py:131 2.298s
    Then I should see the tabular report               # kalite/coachreports/features/steps/coachreports.py:204
Then I should see the tabular report               # kalite/coachreports/features/steps/coachreports.py:204 5.510s
```

And my suspicion is that the wait to find the element is not failing because all this is taking place on the _same_ page, so the element _exists_ but hasn't been updated/refreshed according to new conditions stipulated in the "given" steps. The other failures are similar -- the element being searched for _could_ exist, but may not reflect new conditions.  I assumed that `WebDriver.get` was the culprit, but if it does in fact wait for a page load then perhaps not. Unless it doesn't load the page again if the url is the same?